pkg/utils/urls/urls.go

```diff
@@ -25,6 +25,10 @@ const (
 	Client        // Client == 1
 )
 
+func GenerateFileShareURL(urlType int, tenantId string, in ...string) string {
+        return generateURL("file/shares", urlType, tenantId, in...)
```

Comment:
will generate URL report some error? If yes, then we should have return codes, to the callee.